frontend/src/app/testResults/__snapshots__/TestResultPrintModal.test.tsx.snap

```diff
@@ -547,59 +566,62 @@ Object {
               <h2>
                 More information
               </h2>
-              <p>
-                Contact your health care provider to decide if additional testing is needed, especially if you experience any of these symptoms:
-              </p>
-              <ul
-                class="sr-multi-column"
+              <div
+                class=""
               >
-                <li>
-                  Shortness of breath or difficulty breathing
-                </li>
-                <li>
-                  Fatigue
-                </li>
-                <li>
-                  Muscle or body aches
-                </li>
-                <li>
-                  Headache
-                </li>
-                <li>
-                  Loss of taste or smell
-                </li>
-                <li>
-                  Sore throat
-                </li>
-                <li>
-                  Congestion or runny nose
-                </li>
-                <li>
-                  Nausea or vomiting
-                </li>
-                <li>
-                  Diarrhea
-                </li>
-              </ul>
-              <p>
-                For more information go to 
-                <a
-                  href="https://www.cdc.gov/"
-                  rel="noopener noreferrer"
-                  target="_blank"
-                >
-                  CDC.gov
-                </a>
-                 or call 1-800-CDC-INFO (1-800-232-4636). Use the 
-                <a
-                  href="https://www.cdc.gov/coronavirus/2019-ncov/your-health/covid-by-county.html"
-                  rel="noopener noreferrer"
-                  target="_blank"
+                <p>
+                  Contact your health care provider to decide if additional testing is needed, especially if you experience any of these symptoms:
+                </p>
```

Comment:
Looking at these screenshots, it seems like our link to the CDC's county check tool is gone - is that right or am I just missing something?

Reply:
OOO that's a good catch! Let me investigate!!

Reply:
Upon investigation, yes that is the correct behavior. That county check only should show up for POSITIVE covid results.